src/oop_ext/interface/_interface.py

```diff
@@ -879,7 +951,7 @@ def DeclareClassImplements(class_, *interfaces):
         raise
 
 
-def _GetMROForOldStyleClass(class_):
+def _GetMROForOldStyleClass(class_: Type) -> List[Type]:
```

Comment:
Mypy is complaining about the usage of `Type`
_interface.py:954: error: Missing type parameters for generic type "Type"

Reply:
Again, not seeing this. Please post more details on how you are getting this error.

Reply:
Oh sorry, this specific message was not intended to be sent.
I was initially writing this as a question because I was confusing about some concepts about `Type`  and sent it by mistake.